trade-module/src/main/java/io/edpn/backend/trade/adapter/persistence/filter/MybatisLocateSingleHopTradeFilter.java

```diff
@@ -0,0 +1,34 @@
+package io.edpn.backend.trade.adapter.persistence.filter;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@EqualsAndHashCode
```

Comment:
Are _all_ of those _really_ necessary? If it has a builder and an all args c'tor, what are setters and a no-args c'tor for?

Reply:
Yes. Mybatis